scripts/test-dssspell-forge.sh

```diff
@@ -21,7 +21,7 @@ export DAPP_LIBRARIES="src/DssSpell.sol:DssExecLib:$DSS_EXEC_LIB"
 export DAPP_BUILD_OPTIMIZE=0   # forge turns on optimizer by default
 
 if [[ -z "$MATCH" && -z "$BLOCK" ]]; then
-    forge test --fork-url "$ETH_RPC_URL" -vvv
+    forge test --fork-url "$ETH_RPC_URL"
```

Comment:
why'd you revert this?

Reply:
https://github.com/makerdao/spells-mainnet/pull/250#discussion_r891699803